code/_compile_options.dm

```diff
@@ -45,4 +45,15 @@
 
 #ifdef CITESTING
 #define TESTING
+#endif
+
+#ifdef TGS
+// TGS performs its own build of dm.exe, but includes a prepended TGS define.
+#define CBT
+#endif
+
+#if !defined(CBT) && !defined(SPACEMAN_DMM)
+#warn Building with Dream Maker is no longer supported and will result in errors.
```

Comment:
You need to build the game using build.bat in tools folder. At least do it once to build tgui interfaces.

Reply:
> what are the consequences of this? Will we need to switch to a new editor?
> Why exactly is it no longer supported

Because TGUI need CBT compiling. CBT can't compile in DM. For compiling CBT you need run build.bat or use VSCode CTRL+SHIFT+B